pkg/virt-operator/install-strategy/security.go

```diff
@@ -0,0 +1,104 @@
+package installstrategy
```

Comment:
Maybe `ssc.go`? Would help to distinguish `rbac` from `ssc`, since both are related  to `security`.